includes/class-activitypub.php

```diff
@@ -100,17 +101,16 @@ public static function render_activitypub_template( $template ) {
 			return $template;
 		}
 
+		self::add_headers();
```

Comment:
Should this be called after the `is_activitypub_request()` check?

Reply:
this is the header that should be added to show apps where to find the JSON version, so there is no problem with adding it to all headers